src/resources/postcss/components/_search.pcss

```diff
@@ -48,18 +48,19 @@
 	.tribe-common-c-search {
 		@media (--viewport-medium) {
 			display: flex;
+			flex: 1 1 100%;
 		}
 	}
 
 	.tribe-common-c-search__input-group {
 		@media (--viewport-medium) {
 			background-color: var(--color-background);
-			border: 1px solid var(--color-border-default);
 			border-right: none;
 			display: flex;
 			flex: auto;
-			max-width: 645px;
 			padding: 8px 0;
+			align-items: center;
+			justify-content: center;
```

Comment:
curious why you decided to center justify the input group contents

Reply:
Snap, was testing something and it ended up there.